libraries/core/commands/shareItem.js

```diff
@@ -0,0 +1,26 @@
+import AppCommand from '../classes/AppCommand';
+
+/**
+ * Sends a shareItem command to the app.
+ * @param {Object} params The command paramters.
+ * @param {string} params.title The product title
+ * @param {string} params.imageURL The products image url to be loaded
+ * @param {string} params.deepLink The products deep link
+ */
+export default (params) => {
+  const merged = {
+    title: '',
+    imageURL: '',
+    deepLink: '',
+    ...params,
+  };
+
+  if (merged.deepLink) {
+    merged.deepLink = merged.deepLink.replace(/(\?|&)shopgate_redirect=1/i, '');
```

Comment:
Is the regex really required here? Could we simplify it?

Reply:
This regex was in our legacy implementation for the appCommand and I believe it covers an instance of shopgate redirect  in the deep link url. Not sure if still necessary.

Reply:
OK